### ISSUE TITLE
Quickstart Card module follow-up tasks

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#377]
On 2020-11-13 we merged an initial version of the Quickstart cards module containing card field plugins and a template but we identified several follow-up improvements that are still needed.

## Feedback from 2020-11-13 Meeting:

- [x] Use Standard format by default for body (fix now)
- [ ] How do we delete cards? (investigate now)
- [x] Move background color selection to top
- [x] Move media subfield to second top (fix now)
- [x] Change link_uri label to Card Link URL (fix now)
- [ ] Can we remove blank card that shows up at bottom? (investigate now)
- [x] 2-step edit process for cards? (follow-up)

**Comment by smayden** [comment]
Some additional comments from reviewing PR #634:

-When I click edit on cards (actually all paragraphs) it seems to jump me down to the bottom of the edit area. I think this will confuse content editors
-We might consider making the edit/collapse buttons for each card in the upper right-hand corner to be consistent with where these buttons appear elsewhere on the edit screen.
-Is there a way to make the whole card clickable? I couldn't figure out how to do that (maybe we decided not to do that?)
-Should we consider being able to change the color of the link button? I made a card with red background and button blends with background (could button switch to reverse similar to how text does based on color?)
-Is there a reason we default to basic html rather than the standard text format? (I probably missed or forgot the conversation...) I was trying to make a card with my own button in the card body and was unable to do so until I switched to standard (and agreed to the scary warning).
